### PR TITLE
init to validate staging workflow

### PR DESCRIPTION
**Pull request by nutshell113**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -66,6 +64,18 @@ executors:
     resource_class: large
 
 commands:
+  setup-node-version:
```
Comment (webberwang) [inline .circleci/config.yml:67]: I would keep this inside setup-env, same command but should work for both executor

File: .circleci/config.yml [hunk 2]
```diff
@@ -158,6 +159,24 @@ commands:
           mentions: $SLACK_USER
 
 jobs:
+
+  install-and-build-workspace-staging:
```
Comment (webberwang) [inline .circleci/config.yml:163]: I'd like to keep only `install-and-build-workspace` for both

**Review by webberwang — CHANGES_REQUESTED** [review]
Also see if you can rebase and figure out the naming convention

File: .circleci/config.yml (hunk 1 above)
Comment (nutshell113) [reply .circleci/config.yml:67]: They're different ENVs(diff docker images), we must separate them to diffrent job to let it work .

**Comment by nutshell113** [comment]
This PR is too mess, let replace it by https://github.com/codelab-ai/codelab.ai/pull/245

**Pull request closed without merging**